apps/server/test/globalSetup.ts

```diff
@@ -3,7 +3,7 @@ import { MongoMemoryServer } from 'mongodb-memory-server-global';
 export = async function globalSetup() {
 	const instance = await MongoMemoryServer.create({
 		binary: {
-			version: '5.0.26',
+			version: '6.0.15',
```

Comment:
if 6.0.16 if avilebel then use it
```suggestion
			version: '6.0.16',
```